provisioning_templates/snippet/ssh_authorized_keys.erb

```diff
@@ -0,0 +1,38 @@
+<%#
+kind: snippet
+name: ssh_authorized_keys
+model: ProvisioningTemplate
+snippet: true
+%>
+
+<%#
+SSH keys setup snippet
+
+Parameters:
+
+ssh_authorized_keys: public keys to be put in /root/.ssh/authorized_keys
+
+This template sets up SSH keys in any host so that as long as your public
+SSH key is in ssh_authorized_keys, you can SSH into a host.
+%>
+
+<% if !host_param('ssh_authorized_keys').blank? %>
+<% authorized_keys = host_param('ssh_authorized_keys') %>
+<% ssh_user = 'root' %>
+<% ssh_path = "~#{ssh_user}/.ssh" %>
```

Comment:
I think `~#{ssh_user}` is a bashism and probably doesn't work on other shells.

Reply:
Not invented here... would you prefer calling `getent passwd #{ssh_user} | cut -d":" -f 6`?
https://github.com/theforeman/community-templates/blob/03281e9138e178f336958d1ef83739483c048078/provisioning_templates/snippet/create_users.erb#L12

Reply:
I guess it's safe enough and otherwise patches welcome :)